recipe/build.sh

```diff
@@ -2,6 +2,10 @@
 if [ "$target_platform" = "osx-arm64" ]; then
   export CMAKE_OSX_ARCHITECTURES="arm64"
 fi
+# fix scikit-build not finding the cross-compile python headers.
+if [ "$target_platform" != "$build_platform" ]; then
+  crosspython="-DPython_FIND_VIRTUALENV=ONLY"
```

Comment:
this is pretty safe to always specify.

The problem is.... scikit-build doesn't use the new Python stuff, so try `PYTHON` as well as `Python`

Reply:
Hmm, I was under the impression that FindPython.cmake is used directly from cmake. scikit-build does not provide it itself.

And the currently failing search is from CMakelists.txt `find_package(Python COMPONENTS Interpreter Development NumPy REQUIRED)`